app/src/test/java/comcmput301f16t01/github/carrier/UserProfileTests.java

```diff
@@ -125,14 +124,12 @@ public void showUsernameIfyouAreRider() {
     public void showUsernameIfyouAreDriver() {
         Rider rider = new Rider("Sarah");
         Request request = new Request(rider, new Location(), new Location());
-        RequestController rc = new RequestController();
+        UserController uc = new UserController();
         Driver you = new Driver("you");
-        rider.setEmail("email");
-        rider.setPhone("phone");
+        
+        uc.setEmail(rider, "email");
```

Comment:
You never added this rider to the UserController, the controller will never be able to find this user and edit their data. But this is good use of MVC inside your test! You may need more comments for clarity, in general.